scripts/check-mailbox-hooks.sh

```diff
@@ -22,9 +22,15 @@ for chain_dir in chains/*/; do
     metadata_file="$chain_dir/metadata.yaml"
     addresses_file="$chain_dir/addresses.yaml"
 
+    # Check if the chain is an Ethereum protocol type
+    if ! yq e '.deployer.name == "Abacus Works"' "$metadata_file" | grep -q true; then
+        # echo "$chain_name: Not Abacus Works, skipping."
```

Comment:
nit: do we want this? ditto for the other 3 commented out echos below

Reply:
ah rip i thought i added them back again, will update